### PR TITLE
Collection to Markdown

### LINKED ISSUE #29: Convert a collection index to Markdown
As an individual or organization seeking to share extensions of ATT&CK, I want to be able to convert a [collection index](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/master/docs/collections.md#collection-indexes) into a human-readable Markdown representation like [the one seen on attack-stix-data](https://github.com/mitre-attack/attack-stix-data/blob/master/index.md).

[This script has already been implemented](https://github.com/mitre-attack/attack-stix-data/blob/master/util/index-to-md.py), but needs to be ported into this library. A CLI should be provided similar to the existing layer exporter CLI so that users can utilize the script from the command line provided the library is installed.

### PR DESCRIPTION
**Pull request by ArtificialErmine**
Ported Collections to Markdown script, updated setup.py with necessary changes

Closes #29

### CODE REVIEW

File: mitreattack/navlayers/README.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ This folder contains modules and scripts for working with ATT&CK Navigator layer
 | [matrix_gen](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/exporters/matrix_gen.py) | Provides a means by which to generate a matrix from raw data, either from the ATT&CK TAXII server or from a local STIX Bundle. |
 | [svg_templates](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/exporters/svg_templates.py) | Provides a means by which to convert a layer file into a marked up svg file. |
 | [svg_objects](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/exporters/svg_objects.py) | Provides raw templates and supporting functionality for generating svg objects. |
+##### Collections Scripts
+| script | description |
+|:-------|:------------|
+|[col_to_md](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/navlayers/collections/col_to_md.py)| Provides a means by which to convert a collection index into a human-readable markdown file. More information can be found in the corresponding [section](#col_to_md.py) below.|
```
Comment (isaisabel) [inline mitreattack/navlayers/README.md:36]: As mentioned in [this comment](https://github.com/mitre-attack/mitreattack-python/issues/29#issuecomment-900326058), collection features should be in their own submodule, not a part of `navlayers`. In other words they should be in an organizational grouping at the same level as navlayers and attackToExcel.

File: mitreattack/navlayers/README.md [hunk 2]
```diff
@@ -249,3 +253,21 @@ conf.load_from_file(filename="path/to/poster/config.json")
 t2 = ToSvg(domain='mobile', source='local', local='path/to/local/stix.json', config=conf)
 t2.to_svg(layerInit=lay, filepath="demo2.svg")
 ```
+
+## col_to_md.py
```
Comment (isaisabel) [inline mitreattack/navlayers/README.md:257]: `col` is more generally understood to mean "column", not "collection." Should probably be written out fully in the file name.

Comment (isaisabel) [reply mitreattack/navlayers/README.md:257]: Also, this is converting a collection index, not a collection. So even `CollectionToMarkdown` is not technically correct.

**Review by isaisabel — CHANGES_REQUESTED** [review]
Behaviorally looks great. Has some confusing naming of classes and filenames and also appears to be made a sub-module of navlayers even though it has nothing to do with ATT&CK Navigator layers.

File: mitreattack/navlayers/README.md (hunk 1 above)
Comment (ArtificialErmine) [reply mitreattack/navlayers/README.md:36]: My bad - I misunderstood that line. I'll move it, and adjust the documentation accordingly.

File: mitreattack/navlayers/README.md [hunk 3]
```diff
@@ -249,3 +253,21 @@ conf.load_from_file(filename="path/to/poster/config.json")
...
+into a [markdown file]() for easy of use and reference. The CollectionToMarkdown class is very simple, and provides a 
+single method, `index_to_markdown`, which in turn only requires a single parameter - a dictionary representation of the 
+desired index file to convert to markdown. An example of how to use the class, and method, can be found below.
+#### Example Usage
+```python
+import json
+from mitreattack.navlayers import CollectionToMarkdown
+
+with open('input_file.json', 'r') as input_file:
+    with open('output_file.md', 'w') as output_file:
```
[the remark this answers is not in the document]
Comment (ArtificialErmine) [reply mitreattack/navlayers/README.md:268]: Yeah, that makes sense.

File: mitreattack/navlayers/README.md (hunk 2 above)
Comment (ArtificialErmine) [reply mitreattack/navlayers/README.md:257]: Gotcha, I'll rename things to hopefully clear things up a bit.

File: mitreattack/collections/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+##### Collections Scripts
+| script | description |
+|:-------|:------------|
+|[index_to_markdown](https://github.com/mitre-attack/mitreattack-python/blob/master/mitreattack/collections/index_to_markdown.py)| Provides a means by which to convert a [collection index](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/master/docs/collections.md#collection-indexes) into a human-readable markdown file. More information can be found in the corresponding [section](#index_to_markdown.py) below.|
+
+## index_to_markdown.py
+index_to_markdown.py provides the IndexToMarkdown class, which provides a way to transform an existing [collection index file](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/blob/master/docs/collections.md#collection-indexes) 
+into a markdown file for easy of use and reference. The CollectionToMarkdown class is very simple, and provides a 
+single method, `index_to_markdown`, which in turn only requires a single parameter - a dictionary representation of the 
+desired index file to convert to markdown. An example of how to use the class, and method, can be found below.
```
Comment (isaisabel) [inline mitreattack/collections/README.md:14]: The old class name `CollectionToMarkdown` is still referenced in this documentation.

**Review by isaisabel — COMMENTED** [review]
Has an inconsistency in the documentation but apart from that this is perfect and ready to be merged!

File: mitreattack/collections/README.md (hunk 4 above)
Comment (ArtificialErmine) [reply mitreattack/collections/README.md:14]: Thanks for catching that!

**Pull request merged**